projects/addon-commerce/utils/is-card-length-valid.ts

```diff
@@ -20,6 +20,6 @@ export function isCardLengthValid(cardNumber: string): boolean {
         case TuiPaymentSystem.Visa:
             return length > 12 && length < 20;
         default:
-            return false;
+            return length > 8;
```

Comment:
Could you also add tests that fix this bug?

Reply:
Add in new commit

Reply:
What do you think about?

`return length > 8 && length < 20;`

![image](https://user-images.githubusercontent.com/12021443/134180623-e286f763-c87b-4fb2-876e-d068f2302542.png)